CheckValues/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CheckValues
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
```

Comment:
Why?

Reply:
man i didnt even know that was there

Reply:
We use that project for debugging

Reply:
dont you read your pr when creating it ?

Reply:
took it out bro